$OpenBSD: patch-src_map_albersequal_h,v 1.1.1.1 2020/01/23 08:20:43 landry Exp $
../src/map/albersequal.h:28:9: error: expected member name or ';' after declaration specifiers
        double _C;
        ~~~~~~ ^
/usr/include/ctype.h:50:12: note: expanded from macro '_C'
#define _C      0x20


Index: src/map/albersequal.h
--- src/map/albersequal.h.orig
+++ src/map/albersequal.h
@@ -25,7 +25,7 @@ class AlbersEqual : public CT (private)
 
 	double _a2;
 	double _rho0;
-	double _C;
+	double C;
 	double _n;
 	double _e;
 	double _es;
